app/bundles/DynamicContentBundle/EventListener/DynamicContentSubscriber.php

```diff
@@ -108,7 +115,8 @@ public function __construct(
         AuditLogModel $auditLogModel,
         LeadModel $leadModel,
         DynamicContentHelper $dynamicContentHelper,
-        DynamicContentModel $dynamicContentModel
+        DynamicContentModel $dynamicContentModel,
+        ContactTokenReplacer $contactTokenReplacer
```

Comment:
This should be type of `TokenReplacerInterface`, right?

Reply:
I don't think so, because PHPstorm auto create the code and I want param  contactTokenReplacer not just tokenReplacer. It's better readable :) But I can rewrite. It's so big issue?